SwanCustomEnvironments/swancustomenvironments/scripts/makenv.sh

```diff
@@ -4,16 +4,6 @@
 # Copyright CERN
 # This script allows to create an environment to use in notebooks and terminals. The environment contains the packages from a provided repository.
 
-
-# Check if an environment already exists in the session, avoiding multiple environments
```

Comment:
While it makes some sense to stop checking in the caller/backend side (because we basically impose a singleton), I would argue this script should still verify and exit on error. Unless is ok to call makenv.sh multiple times, this should be protected as there could be multiple ways of calling it.

Reply:
We actually were debating about this with @rodrigo-sobral , we see pros and cons in leaving / removing the code. Leaving it means more code to maintain that is actually not used in our workflow. Removing it removes the protection in case someone finds the script and invokes it manually, as you pointed out. I'm fine with leaving the code.

Reply:
I would suggest leaving it then.